### ISSUE TITLE
Automatic startup bug?

### ISSUE THREAD

**Issue by c-sanchez** [localsend/localsend#564]
I enabled LocalSend autostart in Widows 10, and although the application starts, I always have to close and start it again when I want to send something to/from my android phone.
That is, the application must be started manually after the computer starts up.
It's not a major problem for me, but it's a bit annoying when you expect the program/service to be running from the start.
Can anyone confirm if they have already had this same problem?

I think it has already been discussed in some other issue, but it would be nice to have some "persistent notification" in Android, to make sure that the LocalSend service is running, maybe with information such as number of files being sent/received at the moment, etc.

**Comment by Tienisto** [comment]
So autostart is enabled, LocalSend starts automatically but fails to receive/send files?

**Comment by c-sanchez** [comment]
Yes, the phone does not detect the computer and vice versa, until I close and start the program again.
Maybe the LocalSend takes some different ip or something? I have no idea really ;p

**Comment by Tienisto** [comment]
Weird, Does the PC show up on Android when you press the "scan" button in the send tab? Or is a full restart of the app needed?

**Comment by c-sanchez** [comment]
I just added LocalSend back to the list of auto-start programs, restarted my computer and phone and it works as expected.
hahaha I don't know now if I'm blind, I did something wrong before or what, but well, it works without problem now hehe.

I guess this topic can be closed, although in order not to create another one I will comment, in case it was not done before.

Have you noticed that LocalSend takes a while to open when you want to send a file depending on its size?
With files of a few megabytes it opens almost immediately, but when you try to send gibabyte files it takes considerably longer, the larger the file the longer it takes to open the LocalSend interface to select where to send it.
